src/server/system_services/schemas/account_schema.js

```diff
@@ -25,7 +25,7 @@ module.exports = {
 
         // password login
         has_login: { type: 'boolean' },
-        password: { wrapper: SensitiveString }, // bcrypted password
+        password: { wrapper: SensitiveString }, // password
```

Comment:
changing the meaning of a field in schema is breaking backward compatibility.
also notice that the upgrade script will forever delete the password to all users, which means we cannot ever introduce it again... Instead I would prefer to add a new field called `password_hash_type` with enum of ['bcrypt'] which we can also add 'PBKDF2-HMAC-SHA512-ITER100000' or 'argon2'. this way, the upgrade script just needs to fill for every account with password that will also have value for password_hash_type='bcrypt', and our authentication logic can decide to reject any password authentication with bcrypt, since we do not support it anylonger, so every old account will be forced to renew the password to use it - we should set the account.next_password_change = new Date() to force it.